mmdet3d/models/layers/points_sampler.py

```diff
@@ -0,0 +1,217 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import math
+from typing import List
+
+import numpy as np
+import torch
+from mmcv.ops.furthest_point_sample import furthest_point_sample
+from torch import Tensor
+from torch import nn as nn
+
+from mmdet3d.registry import MODELS
+from mmdet3d.utils.typing import InstanceList
+
+
+@MODELS.register_module()
```

Comment:
Suggest using `TASK_UTILS` registry since this class doesn't have any parameterized modules.
Also, suggest placing this file under `mmdet3d/models/task_modules`